src/components/CardEdition/CardEdition.test.tsx

```diff
@@ -0,0 +1,205 @@
+                blockName="more-link"
+                decorative={true}
+                modifiers={["left"]}
+                iconRotation={IconRotationTypes.rotate0}
+              ></Icon>
+              Download
+            </Link>
+          </div>
+        </div>
+      }
+    ></CardEdition>
+  );
+
+  const cardWithNoImage = (
+    <CardEdition
+      id="card#1"
+      className="edition-card"
+      heading={
+        <Heading
+          level={2}
+          id="editioncardheading1"
+          url="#edition-link"
+          text={"2004 Edition"}
+        />
+      }
+      ctas={
+        <div className="edition-card__ctas">
+          <Link type={LinkTypes.Button} href="blah">
+            Read Online
+          </Link>
+          <div className="edition-card__download">
+            <Link href="#url" type={LinkTypes.Action}>
+              <Icon
+                name={IconNames.download}
+                blockName="more-link"
+                decorative={true}
+                modifiers={["left"]}
+                iconRotation={IconRotationTypes.rotate0}
+              ></Icon>
+              Download
+            </Link>
+          </div>
+        </div>
+      }
+    >
+      middle column content
+    </CardEdition>
+  );
+
+  it("Generates a Card with a header, footer, image, middle content, and CTAs", () => {
+    const card = Enzyme.mount(regularCard);
+    expect(card.find(".card-edition__heading")).to.have.lengthOf(1);
+    expect(card.find(".card-edition__image")).to.have.lengthOf(1);
+    expect(card.find(".card-edition__content")).to.have.lengthOf(1);
+    expect(card.find(".card-edition__ctas")).to.have.lengthOf(1);
+    expect(card.find(".card-edition__footer")).to.have.lengthOf(1);
+  });
+
+  it("Generates a Card with variable data", () => {
+    const card = Enzyme.mount(cardWithExtendedStyles);
+    expect(card.find("h2")).to.have.lengthOf(1);
+    expect(card.find(".card-edition__content").find("div")).to.have.lengthOf(4);
+    expect(card.find(".card-edition__ctas").find("a")).to.have.lengthOf(2);
+  });
+
+  it("Generates a card without a CTA block if one isn't provided", () => {
+    const card = Enzyme.mount(cardWithNoCTAs);
+    expect(card.find(".card-edition__heading")).to.have.lengthOf(1);
+    expect(card.find(".card-edition__image")).to.have.lengthOf(1);
+    expect(card.find(".card-edition__content")).to.have.lengthOf(1);
+    expect(card.find(".card-edition__ctas")).to.have.lengthOf(0);
+  });
+
+  it("Generates a card without a content block if one isn't provided", () => {
+    const card = Enzyme.mount(cardWithNoContent);
+    expect(card.find(".card-edition__heading")).to.have.lengthOf(1);
+    expect(card.find(".card-edition__image")).to.have.lengthOf(1);
+    expect(card.find(".card-edition__content")).to.have.lengthOf(0);
+    expect(card.find(".card-edition__ctas")).to.have.lengthOf(1);
+  });
+
+  it("Generates a card without an image block if no image is provided", () => {
```

Comment:
Can you add similar tests for when no header or footer are added?

Reply:
`CardEdition` will be deprecated, so I am not going to make any changes to the component.